src/components/Folder.vue

```diff
@@ -20,18 +20,18 @@ const toggle = () => {
 </script>
 
 <template>
-  <div class="tl-mb-4">
+  <div class="tl-mb-4 tl-w-full tl-border-x-none tl-border-solid tl-border-1 tl-border-primary">
```

Comment:
![Screenshot 2023-12-18 at 11 06 38](https://github.com/Tresjs/leches/assets/4699008/d77cc03c-c094-4f27-be5d-6305c236b1bd)

Folders look different now, they had a subtle gray background and no border before.

Reply:
Hey there! I tried to emulate the [tweak pane folder](https://tweakpane.github.io/docs/ui-components/#folder) component, which it clearly denoted the separation between one folder and another. I don't mind turning it back or adapting the old styles (which I removed without asking, my bad) to the new component =] Let me know!